pkg/ebpf/netlink/conntracker.go

```diff
@@ -278,10 +293,16 @@ func (ctr *realConntracker) compact() {
 	ctr.Lock()
 	defer ctr.Unlock()
 
+	gen := getCurrentGeneration(compactInterval, time.Now().UnixNano())
+
 	// https://github.com/golang/go/issues/20135
-	copied := make(map[connKey]*IPTranslation, len(ctr.state))
+	copied := make(map[connKey]*connValue, len(ctr.state))
 	for k, v := range ctr.state {
-		copied[k] = v
+		if v.expGeneration != gen {
```

Comment:
shouldn't this be `v.expGeneration <= gen`?

Reply:
It should not, because they roll over at 255

Reply:
oh, right 🤦‍♂